src/test/tests-setup.js

```diff
@@ -29,5 +29,4 @@ setConfig(config);
 
 process.on("unhandledRejection", (reason, p) => {
   console.log("Unhandled Rejection at:", p, "reason:", reason);
-  throw reason;
```

Comment:
i think we want the tests to fail :/

Reply:
the test fails anyway, but this exits the test runner, making it hard to see where the test failed

Reply:
i see. okay